src/components/AddItems.jsx

```diff
@@ -0,0 +1,73 @@
+import { useCallback } from 'react';
+import { useStateWithStorage } from '../utils';
+import { addItem } from '../api';
+import TextInputElement from './TextInputElement';
+import RadioInputElement from './RadioInputElement';
+
+const nextPurchaseDate = {
+	soon: 7,
+	kindOfSoon: 14,
+	notSoon: 30,
+};
+
+const radioInputOptions = {
+	soon: ['soon', nextPurchaseDate.soon, 'Soon'],
+	kindOfSoon: ['kindofsoon', nextPurchaseDate.kindOfSoon, 'Kind of soon'],
+	notSoon: ['notsoon', nextPurchaseDate.notSoon, 'Not Soon'],
+};
+
```

Comment:
I like what I'm seeing! However, the structure of the object seems a bit more complex than necessary. How about simplifying it to an array of objects with just `label` and `value` key-value pairs? This could make it easier to read and manage.